### PR TITLE
Adds fugu gland functionality to humans

### PR DESCRIPTION
**Pull request by ArcLumin**
Title. Doesn't boost their physical attack but buffs their health more

### CODE REVIEW

**Comment by ghost** [comment]
Arc, dearie, I truly hope this is a meme PR.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -637,3 +637,14 @@
 		A.environment_smash += 2
 		user << "<span class='info'>You increase the size of [A], giving it a surge of strength!</span>"
 		qdel(src)
+	if(proximity_flag && istype(target, /mob/living/carbon))
+		var/mob/living/carbon/H = target
+		if(H.buffed || H.buffed)
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:642]: I laughed more than I should have.

**Comment by ArcLumin** [comment]
yup. I'm literally shotgunning PRs

**Comment by blobbernaut** [comment]
+2

**Comment by ArcLumin** [comment]
head approval done, FASTMERGE NO CODECHECK

**Pull request closed without merging**